src/types/acl.graphql

```diff
@@ -33,6 +33,27 @@ type Acl {
   acl: JSON @deprecated(reason: "Deprecating in favor of `groupPermissions`, `legacyGuid`, `catalogItemIdentity`.")
 }
 
+type GroupPermissionList {
```

Comment:
Thinking this through, I don't think we'll ever need to know the number of group permissions an ACL has, we could probably just make this a [GroupPermission]